setup.sh

```diff
@@ -9,6 +9,7 @@ sudo -n dnf install -y pipenv
 cd /vagrant
 # Install dependencies with Pipenv
 pipenv sync --dev
+pipenv install pillow
```

Comment:
I think you should upstade the Pipefile and Pipefile.lock
And add it once you made pipenv lock, please review tool introduction challenge

Reply:
Line removed from setup.sh

Reply:
Please apply pipenv lock
And add Pipfile.lock to PR

Reply:
I already updated the Pipfile and Pipfile.lock files (I think it did it automatically when I installed Pillow), are more changes needed to be applied?